### ISSUE TITLE
I can't get the result as you shown

### ISSUE THREAD

**Issue by HonkeeChan** [MarkPrecursor/SRCNN-keras#5]
the result you shown in this repository seems very good. Thanks for sharing the code.
But I use the learning rate 0.0003 and train 200 epoch, get the mse loss is 0.0011.
but I finaily get bad result:

Epoch 200/200
36400/36400 [==============================] - 4s - loss: 0.0011 - mean_squared_error: 0.0011
bicubic:
27.0124821041
SRCNN:
12.1705450186

what's can I do for this? Thank you very much.
